qownnotes/snapcraft.yaml

```diff
@@ -40,6 +38,9 @@ parts:
       - libqt5svg5
       - libqt5network5
       - libqt5xml5
+      - libqt5xmlpatterns5
       - libqt5printsupport5
       - libqt5declarative5
-    after: [qt5conf]
+    stage:
+      - -usr/share/pkgconfig
+    after: [desktop/qt5]
```

Comment:
Interesting, that worked! I will try to build with that locally and on Launchpad too! Thanks a lot, @dholbach!